server/src/server.js

```diff
@@ -205,6 +209,20 @@ app.get("/content/:id/:domain", function (req, res) {
   });
 });
 
+app.get("/images/:id/:domain/:clipId", function (req, res) {
+  Shot.prototype.Clip.getRawBytesForClip(
```

Comment:
Why is this attached to the prototype?  Especially since it doesn't seem to be a method.  I think it would be better just to add to Shot directly (i.e., `Shot.getRawBytesForClip(...)`)